### PR TITLE
INFRA-8081 Create a new autorecycle lambda

### PR DESCRIPTION
**Pull request by jonnyshaw89**
migrate code from monitor-autorecycle-lambda

### CODE REVIEW

File: .major-version [hunk 1]
```diff
@@ -0,0 +1 @@
+0
```
Comment (dflook) [inline .major-version:1]: What is this?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,40 @@
+TERRAFORM_VERSION = 1.2.8
```
Comment (dflook) [inline Makefile:1]: Can we update this

**Review by dflook — APPROVED** [review]

File: .major-version (hunk 1 above)
Comment (jonnyshaw89) [reply .major-version]: No idea. took it from one of the others. deleted

File: Makefile (hunk 2 above)
Comment (jonnyshaw89) [reply Makefile:1]: im not sure....

Comment (jonnyshaw89) [reply Makefile:1]: All the projects on the shared pipeline specify the same version

**Review by dflook — APPROVED** [review]

**Pull request merged**